### PR TITLE
The 0.10.x series should publish 0.10.x artifacts

### PR DESCRIPTION
**Pull request by adelbertc**

### CODE REVIEW

**Comment by adelbertc** [comment]
@timperrett Do we need to do anything else to get CI/CD setup for 0.10.x?

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ steps:
     concurrency: 1
     concurrency_group: primary
     timeout_in_minutes: 45
-    branches: master
+    branches: "master 0.10.x"
```
Comment (timperrett) [inline .buildkite/pipeline.yml]: Is this the correct syntax? I thought it was an array, not space delimited

Comment (adelbertc) [reply .buildkite/pipeline.yml]: I was surprised to, but according to https://buildkite.com/docs/pipelines/branch-configuration#branch-pattern-examples it is a space separated string

**Comment by timperrett** [comment]
The only thing that is needed to build this is to have the CI droplet running, which i can do tomorrow night when I get access to a computer

**Comment by adelbertc** [comment]
@timperrett Sweet! One more thing - I assume the versions that are actually published use the MAJOR.MINOR from `version.sbt` but the patch just uses `$BUILD_NUMBER` or whatever. In that case is the `$BUILD_NUMBER` shared across branches? Or is it going to start trying to publish from `0.10.{0, 1, ...}`? If it starts over we're going to hit some duplicate artifact versions since at the very least we have an existing `0.10.63` or something.

**Comment by timperrett** [comment]
It’s monotonic for the build pipeline, so it’s unique across scm sources, so there shouldn’t be any issues at all.

**Comment by adelbertc** [comment]
@timperrett OK so the next build of 0.10.x branch will probably attempt to publish 0.10.64+ or something ?

**Comment by timperrett** [comment]
Yep, the numbers won’t be sequential of course, but greater numbers will always trump lesser numbers

**Pull request merged**